Makefile

```diff
@@ -38,8 +38,11 @@ hifidata:
 
 ## nanodata:    download Oxford Nanopore test data for test suite
 nanodata:
-	curl -L -o yeat/tests/data/ecolk12mg1655_R10_3_guppy_345_HAC.fastq https://figshare.com/ndownloader/files/21623145
-	gzip yeat/tests/data/ecolk12mg1655_R10_3_guppy_345_HAC.fastq
+	curl -L -o yeat/tests/data/ecolk12mg1655_R10_3_guppy_345_HAC.fastq.gz https://figshare.com/ndownloader/files/21623145
```

Comment:
Realized here that the link downloads a compressed `.gz` fastq file. No need to gzip the fastq file again!